### ISSUE TITLE
Possible to add telemetry server?

### ISSUE THREAD

**Issue by kallestrop** [truckersmp-cli/truckersmp-cli#157]
I tried to installing Funbits Windows ets2-telemetry-server https://github.com/Funbit/ets2-telemetry-server in the wine prefix but it doesn't work. There is also a Python telemetry server available at https://github.com/thomasa88/pyets2_telemetry_server which is only for the Linux version of ETS2.
The author noticed in his post in the SCS forum https://forum.scssoft.com/viewtopic.php?f=34&t=275391  that it is theoretical possible to port his plugin to Windows. Maybe it is somehow possible to add a ported Python version to truckersmp-cli.

**Comment by Lucki** [comment]
Your first guess was correct: The server has to run in wine in the same prefix. Unfortunately it doesn't work in wine and crashes right away.

I'm not sure how portable the python plugin is or how similar the sdk in wine behaves that a port would be easily doable. This probably needs some evaluation from @kakurasan.
Even if both apply here I don't think we're going to include it directly but if it's fully working we can provide an easy way to start it beforehand.

**Comment by kakurasan** [comment]
I tried to use "pyets2_telemetry_server" but it seems that [pyets2_telemetry](https://github.com/thomasa88/pyets2_telemetry) is not compatible with ETS2/ATS 1.39.

First I built it with SCS SDK 1.12 (against Python 3.8.6) but Linux version of ETS2 1.39 didn't detect it.

Next I built SDK example `telemetry_position` (in SCS SDK archive) and ETS2 1.39 showed dialog after starting.

So I guess that `pyets2_telemetry` needs to migrate to SCS SDK 1.12.

I would try to use Windows version of ETS2/ATS *when it's done*.
